integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/WithKyuubiServerOnKubernetes.scala

```diff
@@ -55,3 +72,52 @@ trait WithKyuubiServerOnKubernetes extends WithKyuubiServer {
 
   def getMiniKubeApiMaster: String = miniKubernetesClient.getMasterUrl.toString
 }
+
+class KyuubiOperationKubernetesClusterSuite
+  extends WithKyuubiServerOnKubernetes {
+  private def sessionManager: KyuubiSessionManager =
+    server.backendService.sessionManager.asInstanceOf[KyuubiSessionManager]
+
+  test("KyuubiOnKubernetesWithClusterSparkTestsSuite") {
```

Comment:
this test is for spark cluster mode ? not see the deploy mode config in batch request..

Reply:
shall we add a test for both deploy mode ?

Reply:
In theory, we have no problem using JPS Application Operation to deal with  Client Mode, after all, there are JPS Application Operation tests. But just to be careful, I'll add Client Mode Test later.